module/VuFind/src/VuFind/Sitemap/Generator.php

```diff
@@ -483,31 +565,37 @@ protected function buildIndex($totalPages)
 
             // Add a <sitemap /> group for a static sitemap file.
             // See sitemap.ini for more information on this option.
-            if (isset($this->config->SitemapIndex->baseSitemapFileName)) {
-                $baseSitemapFile = $this->config->Sitemap->fileLocation . '/' .
-                    $this->config->SitemapIndex->baseSitemapFileName . '.xml';
+            $baseSitemapFileName = $this->config->SitemapIndex->baseSitemapFileName
```

Comment:
It looks like the .xml extension has been lost here. Is that intentional? I realize that it may be more flexible to not force the extension, but changing this behavior will break existing configurations (unless the extension is still getting added somewhere and I'm just overlooking it).

Reply:
Oops, no. Fixed.